cloud/deployment/fromfile/fromfile.go

```diff
@@ -184,10 +184,12 @@ func CreateOrUpdate(inputFile, action string, client astro.Client, coreClient as
 // It returns an error if getting default options fail.
 // It returns an error if worker-queue options are not valid.
 // It returns an error if node pool id could not be found for the worker type.
-func getCreateOrUpdateInput(deploymentFromFile *inspect.FormattedDeployment, clusterID, workspaceID, action string, existingDeployment *astro.Deployment, nodePools []astrocore.NodePool, client astro.Client) (astro.CreateDeploymentInput, astro.UpdateDeploymentInput, error) {
+func getCreateOrUpdateInput(deploymentFromFile *inspect.FormattedDeployment, clusterID, workspaceID, action string, existingDeployment *astro.Deployment, nodePools []astrocore.NodePool, client astro.Client) (astro.CreateDeploymentInput, astro.UpdateDeploymentInput, error) { //nolint
```

Comment:
nit: I think at some point we might have to move to using a struct instead of accepting such a large number of inputs. Also might have look at breaking down such large functions, since we have already put linting exception at few places

Reply:
I agree. This has been on my mind too.